src/features/iterator-helpers.md

```diff
@@ -0,0 +1,184 @@
+---
+title: 'Iterator helpers'
+author: 'Rezvan Mahdavi Hezaveh'
+avatars:
+  - 'rezvan-mahdavi-hezaveh'
+date: 2024-03-13
+tags:
+  - ECMAScript
+description: 'Interfaces that help with general usage and consumption of iterators.'
+tweet: ''
+---
+
+V8 shipped a collection of new methods on Iterator prototype (*Iterator helpers*) that help in general use of iterators in v12.2. The introduced helpers are as follow:
```

Comment:
The usual format for feature blog posts is to introduce the feature and how it's useful in the beginning, then at the very end have a section about its availability (in this case, how it's already shipped in v12.2).